packages/server/src/api/controllers/application.ts

```diff
@@ -208,7 +208,7 @@ export async function fetchAppPackage(ctx: UserCtx) {
   let application = await db.get(DocumentType.APP_METADATA)
   const layouts = await getLayouts()
   let screens = await getScreens()
-  const license = await licensing.getLicense()
+  const license = await licensing.getCachedLicense()
```

Comment:
Seems like we should have a test here that mocks `licensing` and asserts that the relevant `plan.type` (or free when there is no license) has been returned in the response.